makefile

```diff
@@ -153,6 +153,13 @@ $(buildDir)/run-linter:scripts/run-linter.go $(buildDir)/.lintSetup
 	go build -o $@ $<
 # end lint setup targets
 
+# generate lint JSON document for evergreen
+generate-lint:$(buildDir)/generate-lint
+	./$<
```

Comment:
explicitly call out the name of the file you're producing, 

also make it a .FORCE/.PHONY or just depend on all the source files (there's a make variable for this already)... right now you won't regenrate files correctly if the source of generate-lint hasn't changed.